rtv/history.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+
+__all__ = ['load_history', 'save_history']
+
+
+def history_path():
+    """
+    Create the path to the history log
+    """
+    HOME = os.path.expanduser('~')
+    XDG_CONFIG_HOME = os.getenv('XDG_CACHE_HOME', os.path.join(HOME, '.config'))
+    path = os.path.join(XDG_CONFIG_HOME, 'rtv', 'history.log')
+    return path
+
+
+def load_history():
+    """
+    Load the history file into memory if it exists
+    """
+    path = history_path()
+    if os.path.exists(path):
+        with open(path) as history_file:
+            return set([line.replace('\n', '') for line in history_file])
```

Comment:
You can use a set comprehension here. It's a small change but I think it looks a little cleaner and saves an intermediate step.

```python
return {line.strip() for line in history_file}
```